### DISCUSSION TITLE
Al Dente continuously needs to be restarted, and sometimes reinstalled

### DISCUSSION THREAD

**Discussion by PipeDream92** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#873] [Q&A]
I've been having issue with Al Dente working properly on my M1 Mac. It installs without issue, and all the modes work as they should. However, after I restart the computer, or even after playing a game via the Epic Games installer, Al Dente stops working. I've managed to fix this reinstalling the helped usually. At one point, I actually had to remove and reinstall. 

Just today, for example, I was on the Mac in the morning, shut the Mac down, unplugged it (as Al Dente will not be preventing it from charging to 100% when shut down from my understanding). When I turned the Mac back on this evening, I was greeted by the icon showing 0% charged. I reinstalled the helper, and it suddenly stated 88%, which is what it should be reading. I don't know what I am doing wrong here. Is there some security or app setting that will make this work? If I have to reinstall every day...it isn't worth the $30.

**Comment by MatthiasKerbl** [comment]
Hi @PipeDream92 ,

Sorry for the inconvenience. We are currently investigating this issue and will try to fix it as soon as possible.

Cheers,
Matthias

**Reply by PipeDream92** [reply]
I have discovered, following uninstalling and reinstalling the app, that I had tried "Open using Rosetta", but had not turned it off. I do not having it running using Rosetta now...and this is where my other issue exists (which "Open using Rosetta" had fixed). For every restart, the password is required to initialize the app. Is there another workaround to mend this problem? Requiring my password twice just to fully boot up my Mac is not considerably convenient.
